Lib/test/test_pickle.py

```diff
@@ -1,18 +1,21 @@
 from _compat_pickle import (IMPORT_MAPPING, REVERSE_IMPORT_MAPPING,
                             NAME_MAPPING, REVERSE_NAME_MAPPING)
 import builtins
-import pickle
-import io
 import collections
+import contextlib
+import io
+import pickle
 import struct
 import sys
+import tempfile
 import warnings
 import weakref
 
 import doctest
 import unittest
+from textwrap import dedent
```

Comment:
Put this one after `import weakref`. I think the imports were sorted between stdlib ones and testing ones (hence the blank line at line 13)